### PR TITLE
Fixed typos in readme.md file and added CONTRIBUTING.md file.

### LINKED ISSUE #64: Fix Typos in Readme and add CONTRIBUTING.md file
### Describe the bug

1. Spelling mistakes and some commas are missing in the readme file.
2. Missing link to the contributing guidelines.
3. Add CONTRIBUTING.md file in .github folder

### Screenshots
Editing open-source-programs-web_README md at develop · anitab-org_open-source-programs-web - Google Chrome 23-01-2021 12_46_14

### PR DESCRIPTION
**Pull request by INNOVATIVEGAMER**
### Description

1.Fixed the contributing spelling
2.Added contrinuting.md file
3.Added the contributing guidelines link

Fixes #64 

### Type of Change:
- Documentation

### Checklist:
- My PR follows the style guidelines of this project
- I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by INNOVATIVEGAMER** [comment]
@isabelcosta Please review this file.
Do the documentation PR also needs 2 reviews ? :confused:

**Comment by isabelcosta** [comment]
@INNOVATIVEGAMER yes :) If you could reach out to the community to get more reviews that would be amazing!

**Review by isabelcosta — DISMISSED** [review]
Thank you for this contribution @INNOVATIVEGAMER 🙌

**Comment by devkapilbansal** [comment]
@INNOVATIVEGAMER we already have a commit message guideline( Which you link in Contributing.md file) Please follow it. Feel free to change the title to PR to be more descriptive.

**Comment by devkapilbansal** [comment]
Also, issue #64 is not related to adding a contribution guidelines file. Please edit the issue for every change this PR contains.

**Comment by isabelcosta** [comment]
Agree with @devkapilbansal points :) thank you for providing your input!

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Contributing Guidelines
+
+-   You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Bridge in Tech stream is _blank_.
```
Comment (gaurivn) [inline .github/CONTRIBUTING.md]: Pls make the necessary changes - 
1. Bridge in Tech should be Open Source Programs instead.
2. Add "Open Source Programs stream is [#open-source-progs](https://anitab-org.zulipchat.com/#streams/237907/open-source-progs)." instead.

Thank you, do make necessary changes in the issue  - add contributing.md

**Review by gaurivn — CHANGES_REQUESTED** [review]

**Comment by INNOVATIVEGAMER** [comment]
@gaurivn Fixed the changes you mentioned in the CONTRIBUTING.md file.

**Comment by INNOVATIVEGAMER** [comment]
@isabelcosta and @devkapilbansal Fixed the changes you mentioned. 
There are 2 commits now.
You can squash and merge them as one commit. I will reset my fork to head commit of the upstream/develop.

Sorry for the commits title , I am new to this repo and as this is just a typo fix in readme so silly mistakes happened.
Thank you !

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # Contributing Guidelines
 
--   You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Bridge in Tech stream is _blank_.
+-   You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Open Source Programs stream is _#open-source-progs_.
```
Comment (gaurivn) [inline .github/CONTRIBUTING.md]: #open-source-progs - missing link to Open Source Programs Zulip stream https://anitab-org.zulipchat.com/#streams/237907/open-source-progs  (as in markdown - [ name ] ( link )   ), no need of italics

Pls add the link, thank you

**Review by gaurivn — CHANGES_REQUESTED** [review]

**Comment by INNOVATIVEGAMER** [comment]
@gaurivn  Fixed the link to zulip community.
Now this typos making me crazy. 😆 
Fixed all or any other changes required ?

**Review by gaurivn — APPROVED** [review]
All good, Thank you for your contribution :confetti_ball: :tada:

**Comment by INNOVATIVEGAMER** [comment]
@isabelcosta and @devkapilbansal Fixed the changes you mentioned. 
There are 3 commits now.
You can squash and merge them as one commit. 
I will reset my fork to head commit of the upstream/develop on my side for future commits so no past commits will be included in other PRs
You can use "Docs :  Fixed typos in readme.md file and added CONTRIBUTING.md file" as your commit title.

Sorry for the commit titles , I am new to this repo and as this is just a typo fix in readme so silly mistakes happened.
Thank you !

**Review by isabelcosta — APPROVED** [review]
Awesome! 👏 thank you for addressing the additional code review comments :)

**Comment by isabelcosta** [comment]
Thank you so much for your inputs @devkapilbansal @gaurivn ! 🎉 
I really appreciate it! 💪

**Pull request merged**